challenges/crossing_ref-zebra/crossing_ref_zebra.py

```diff
@@ -0,0 +1,47 @@
+import sys
+sys.path.append('../../shared')
+import challenge_builder as mrcb
+from tqdm import tqdm
+
+def needs_task(element):
+    return True
+
+
+op = mrcb.Overpass()
+print("Fetching elements from Overpass...")
+
+elements = op.getElementsFromQuery(
+    """
+[out:json][timeout:250];
+area(id:3600051477)->.searchArea;
+nwr["highway"="crossing"]["crossing:markings"="zebra"]["crossing_ref"!="zebra"](area.searchArea);
+nwr["highway"="crossing"]["crossing"="zebra"]["crossing_ref"!="zebra"](area.searchArea);
```

Comment:
We have look at nodes and [ways](https://github.com/openstreetmap/id-tagging-schema/pull/1564/files#diff-ccf8cb8f90cdde1609447c095206391b36850ca4ce1b2b97f90c9f961c7aca1fR15-R27) with different tagging:

```suggestion
node["highway"="crossing"]["crossing:markings"="zebra"]["crossing_ref"!="zebra"](area.searchArea);
node["highway"="crossing"]["crossing"="zebra"]["crossing_ref"!="zebra"](area.searchArea);
way["highway"="footway"]["footway"="crossing"]["crossing:markings"="zebra"]["crossing_ref"!="zebra"](area.searchArea);
way["highway"="footway"]["footway"="crossing"]["crossing"="zebra"]["crossing_ref"!="zebra"](area.searchArea);
```

Reply:
And then there are those other `highway` tags, that also have crossing variants:
* See https://github.com/openstreetmap/id-tagging-schema/pull/1564#discussion_r2106228676
* ~10k https://taginfo.geofabrik.de/europe:germany/tags/cycleway=crossing
* ~8k https://taginfo.geofabrik.de/europe:germany/tags/path=crossing